benchmark/Benchmark/Tomland.hs

```diff
@@ -0,0 +1,39 @@
+module Benchmark.Tomland
+       ( convert
+       , decode
```

Comment:
Let's also reexport and use `parse` from here for consistency across benchmark suits.